pgmpy/inference/ExactInference.py

```diff
@@ -228,6 +230,47 @@ def induced_graph(self, elimination_order):
         >>> inference.induced_graph(['C', 'D', 'A', 'B', 'E'])
         <networkx.classes.graph.Graph at 0x7f34ac8c5160>
         """
+        # If the elimination_order list is incomplete, raise an error
+        if len(elimination_order) < len(self.variables):
```

Comment:
@pratyakshs It's more probable that the arguments would be correct and in that case you will be doing more computation.